src/main/web_src/src/components/fagsystem/tpsf/visning/partials/Relasjoner.tsx

```diff
@@ -18,6 +19,17 @@ type Relasjon = {
 type Person = {
 	ident: string
 	identtype: string
+	fornavn: string
+	mellomnavn?: string
+	etternavn: string
+	kjonn: string
+	alder: number
+	doedsdato: Date
+	foreldreType: string
```

Comment:
Valgfri?

Reply:
Mener denne gir bad request dersom den ikke er satt 😅

Reply:
Åja, tenkte den ikke gjaldt for alle typer relasjoner 😅  Men da må den være med ja 😄